src/software/backend/input/network/networking/network_filter.cpp

```diff
@@ -10,7 +10,7 @@
 // We can initialize the field_state with all zeroes here because this state will never
 // be accessed by an external observer to this class. the getFieldData must be called to
 // get any field data which will update the state with the given protobuf data
-NetworkFilter::NetworkFilter()
+NetworkFilter::NetworkFilter(std::shared_ptr<const RefboxConfig> config)
```

Comment:
Use refbox_config to keep it consistent.

Reply:
changed